torch/_torch_docs.py

```diff
@@ -7999,18 +7999,47 @@ def merge_dicts(*dicts):
            r"""
 svd(input, some=True, compute_uv=True, *, out=None) -> (Tensor, Tensor, Tensor)
 
-This function returns a namedtuple ``(U, S, V)`` which is the singular value
-decomposition of a input real matrix or batches of real matrices :attr:`input` such that
-:math:`input = U \times diag(S) \times V^T`.
+Compute the singular value decomposition of either a matrix or batch of
+matrices :attr:`input`." The singular value decomposition is represented as a
+namedtuple ``(U, S, V)``, such that :math:`input = U \times diag(S) \times
+V^T`, where :math:`V^T` is the transpose of ``V``. If the inputs are batches, then
```

Comment:
"If the inputs are batches..." -> "If :attr:\`input\` is a batch of tensors, then \`\`U\`\`, \`\`S\`\`, and \`\`V\`\` are also batched with the same batch dimensions as :attr:\`input\`."